src/default_environment.rs

```diff
@@ -243,7 +243,7 @@ pub fn default_env() -> Env {
                     let end = end.to_i128().ok_or(RuntimeError::new("Failed converting to `i128`"))?;
                 }
             }
-            
```

Comment:
Fun fact: The only reason I found this is because `cargo fmt` gave an internal error on this line, I'm not sure why it even gave an *internal* error but I guess it doesn't like trailing spaces so much it died from inside lol.